setup.py

```diff
@@ -2,13 +2,13 @@
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
-    
+
 setup(
-    name='QCCalculator',
+    name='qccalculator',
     version='0.0.2',
     packages=find_packages(),
     url='https://gitlab.ebi.ac.uk/walzer/qccalculator',
-    description='Development for python driven QC calculation (QCCalculator) and presentation (QSpector)',
+    description='Development for python driven QC calculation (qccalculator) and presentation (QSpector)',
     long_description=long_description,
     install_requires=[
```

Comment:
Is there a way to import the requirements.txt?

Reply:
Then we would only have to maintain one file in case of dependency changes

Reply:
@mwalzer I have been reading about this and is recommended to use both files. We can improve that in the future.

Reply:
okay